### DISCUSSION TITLE
Complete computer crash (hard shutdown required)

### DISCUSSION THREAD

**Discussion by andreda** [flybywiresim/aircraft#2741] [Help]
Hi folks,

I've noticed that after the latest MSFS patch, I'm getting constant freezes (requiring hard shutdown), especially when I look down to turn the WX radar on.

Is anyone experiencing similar behavior?

**Comment by airfirefighter** [comment]
> Hi folks,
> 
> I've noticed that after the latest MSFS patch, I'm getting constant freezes (requiring hard shutdown), especially when I look down to turn the WX radar on.
> 
> Is anyone experiencing similar behavior?

i hade this issue 1 time after the update.first time a crash on desktop.and i was looking down when this occurred
